lib/src/feedback_widget.dart

```diff
@@ -20,7 +20,8 @@ class FeedbackWidget extends StatefulWidget {
     required this.isFeedbackVisible,
     required this.drawColors,
     required this.mode,
-  })   : assert(
+    this.pixelRatio = 3.0,
```

Comment:
This should be required without a default. That way we don't have to specify the default value a couple times at different places.

Reply:
The idea was to allow the usage of the single component without requiring it, but if you prefer it mandatory for all, sure.